### PR TITLE
Fix issues identified in staging

### PR DESCRIPTION
**Pull request by bewakes**
- Date parsing issue
- Int passing issue
- Null value for national_society_id

Addresses xxxxxx 

## Changes

* Fix Date parsing issue
* Fix Int passing issue
* Fix Null value for national_society_id
* Fix Unused imports and type errors

## Checklist 
Things that should succeed before merging.

- [x] Updated/ran unit tests
- [ ] Updated CHANGELOG.md

## Release

If there is a version update, make sure to tag the repository with the latest version.

### CODE REVIEW

File: dref/assessment_utils.py [hunk 1]
```diff
@@ -89,8 +89,10 @@ def parse_planned_intervention_title(title):
 
 
 def parse_disaster_type(disaster_type):
-    return DisasterType.objects.filter(name__icontains=disaster_type).first()
-
+    try:
+        return DisasterType.objects.filter(name__icontains=disaster_type).first()
+    except ValueError:
```
Comment (bewakes) [inline dref/assessment_utils.py:94]: Is this needed?

File: dref/imminent_utils.py [hunk 2]
```diff
@@ -90,7 +90,10 @@ def parse_planned_intervention_title(title):
 
 
 def parse_disaster_type(disaster_type):
-    return DisasterType.objects.filter(name__icontains=disaster_type).first()
+    try:
+        return DisasterType.objects.filter(name__icontains=disaster_type).first()
+    except ValueError:
```
Comment (bewakes) [inline dref/imminent_utils.py:95]: I don't see any place which can raise ValueError

**Comment by szabozoltan69** [comment]
@bewakes could we merge it?

**Comment by szabozoltan69** [comment]
Though this def occurs 3x:
```
grep -ir "def parse_type_of_onset" 
dref/utils.py:def parse_type_of_onset(onset_type):
dref/imminent_utils.py:def parse_type_of_onset(onset_type):
dref/assessment_utils.py:def parse_type_of_onset(onset_type):
```

**Review by szabozoltan69 — APPROVED** [review]

**Comment by szabozoltan69** [comment]
@k9845 @bewakes @frozenhelium Is anyone still working on this or can I merge it?

**Comment by frozenhelium** [comment]
@szabozoltan69 please go ahead and merge and deploy it!

**Comment by szabozoltan69** [comment]
@frozenhelium deployed to staging

**Comment by szabozoltan69** [comment]
Now the docker log is much nicer.
Only having that package mismatch, "RequestsDependencyWarning: urllib3 (1.26.12) or chardet (5.0.0)/charset_normalizer (2.0.12) doesn't match a supported version"

**Pull request merged**